documents/ms.tex

```diff
@@ -258,24 +259,29 @@ \section{Introduction}
 \begin{equation}
 \label{eq:QP}
 k_{i,j} = A \exp \left[-\frac{(x_i - x_j)^2}{2l^2} -
-\Gamma^2 \sin^2\left(\frac{2\pi}{P}\right) \right].
+\Gamma^2 \sin^2\left(\frac{\vrcomment{\pi(x_i - x_j)}}{P}\right) \right].
 \end{equation}
-It is the product of the SE kernel function, which describes the overall
+\vrcomment{[VR: there was a term missing from the above definition, viz.\ $(x_i - x_j)$ inside the $\sin()$. Without it, the $\sin()$ term reduces to a constant. Also, I changed $2\pi$ to $\pi$: if you define it with $2\pi$ then the period will differ from the physical period by a factor of $2$, given the squared term modifying the sine wave: ${\sin ^2}(\frac{\theta }{2}) = \frac{1}{2} - \frac{1}{2}\cos \theta $. This can be checked by plotting some covariance kernels with large $l$ and noting that peaks appear with expecting spacing $P$ only if you remove the factor of $2$ that originally appeared in the definition.]}   It is the product of the SE kernel function, which describes the overall
```

Comment:
Ruth is using george and the kernel is implemented the way you say there so this is just a typo in the paper and it won't affect the results.